### PR TITLE
Remove myself as codeowner

### PR DESCRIPTION
**Pull request by msiglreith**
I won't have time to do maintainance stuff anytime soon - so removing myself as codeowner.
Thanks everyone & cheers!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -26,8 +26,8 @@
 /src/platform_impl/web              @daxpedda
 
 # Windows
-/src/platform/windows.rs            @msiglreith
-/src/platform_impl/windows          @msiglreith
+# /src/platform/windows.rs            
+# /src/platform_impl/windows
```
Comment (notgull) [inline .github/CODEOWNERS:30]: Put me on as Windows maintainer for now.

**Review by notgull — COMMENTED** [review]
Thanks for everything you've done!

File: .github/CODEOWNERS (hunk 1 above)
Comment (msiglreith) [reply .github/CODEOWNERS]: done, thank you!

**Review by notgull — APPROVED** [review]

**Comment by madsmtm** [comment]
@msiglreith we talked at our meeting today about this, and @MarijnS95 in particular was concerned about the status of `raw-window-metal`? Is that something that would make sense to move to @rust-windowing?

**Comment by msiglreith** [comment]
@madsmtm oh sure I can transfer it to `rust-windowing` if people are fine with it

**Comment by madsmtm** [comment]
Cool, I've opened https://github.com/norse-rs/raw-window-metal/issues/16 for that.

**Pull request merged**